contracts/src/v0.8/shared/test/call/CallWithExactGas.t.sol

```diff
@@ -4,11 +4,13 @@ pragma solidity 0.8.19;
 import {CallWithExactGas} from "../../call/CallWithExactGas.sol";
 import {CallWithExactGasHelper} from "./CallWithExactGasHelper.sol";
 import {BaseTest} from "../BaseTest.t.sol";
-import {GenericReceiver} from "../testhelpers/GenericReceiver.sol";
+import {GenericReceiver, GasConsumer} from "../testhelpers/GenericReceiver.sol";
 
 contract CallWithExactGasSetup is BaseTest {
   GenericReceiver internal s_receiver;
   CallWithExactGasHelper internal s_caller;
+  GasConsumer internal gas_consumer;
```

Comment:
naming should be `s_gasConsumer`

Reply:
What's the reason for this and what is the "s_" supposed to denote?